UI/src/controller/trafficSignsCollection.js

```diff
@@ -34,16 +34,10 @@
         additionalPanels: { values : [45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 147, 146, 145, 144, 143, 142, 141, 140, 139, 138, 148, 149, 150, 151], groupName: 'Lisakilvet'}
     };
 
-    var trafficSignsTurnRestriction = [10, 11, 12];
-    var trafficSignRestriction = [13, 14, 15, 16, 17, 18, 19, 21, 22, 23, 24, 25, 26, 20, 100, 101];
+    var pointToLinearSign = [85, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 100, 101];
 
-    var isTurningRestriction = function(current) {
```

Comment:
Is this function remove ok ?

Reply:
This function here is practically the same as isTrafficSignRestriction and the idea of use is exactly the same too. So I decided to have both on one array/function.

Reply:
After talking with team this is now more clearer.
Nothing changed here